packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -697,16 +701,18 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
           unstable_pinned={unstable_pinned}
           positioningDependencies={[items.length]}
         >
-          <List
-            className={Dropdown.slotClassNames.itemsList}
-            {...accessibilityMenuProps}
-            styles={styles.list}
-            tabIndex={search ? undefined : -1} // needs to be focused when trigger button is activated.
-            aria-hidden={!open}
-            onFocus={this.handleTriggerButtonOrListFocus}
-            onBlur={this.handleListBlur}
-            items={items}
-          />
+          {List.create(list || {}, {
+            overrideProps: {
```

Comment:
Why are all these `overrideProps`? I would expect the callbacks to be in override and to invoke the props[callbacks], but not all of them.